src/main/java/com/olgaboiar/mint/Constants.java

```diff
@@ -0,0 +1,17 @@
+package com.olgaboiar.mint;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+public final class Constants {
+
+    public static final int DEFAULT_PORT = 5000;
+    public static final String DEFAULT_HOST = "localhost";
+
+    public static final String STATUS_CODE_200 = "200 OK";
+    public static final String STATUS_CODE_404 = "404 Not Found";
+
+    public static final String PROTOCOL = "HTTP/1.1";
+
+    public static final String DATE = DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now());
```

Comment:
Since this is a constant, if you started the program yesterday, the `DATE` will always be yesterday. Perhaps you want to constantize the date header `Date: ` or a date format, but not the date itself.

Reply:
removed in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/00c8feb04f6c8f6c322341fc12eb4405e3d64a2e